### PR TITLE
add note about how cascade OGC httpS layers with QGIS Server

### PR DESCRIPTION
**Pull request by gioman**

### CODE REVIEW

**Comment by gioman** [comment]
Possibly (unchecked yet) this is also good to be added to QGIS 3 manual.

**Comment by anneb** [comment]
Some comments:
0. IMHO the documented feature in this PR is very important and should be documented ASAP even if documentation is not (yet) perfect
1. The current version of the PR has a build fail: "Title underline too short"
2. The new comment is now at the bottom of the documentation page. Shouldn't the HOME FcgidInitialEnv configuration be included by default?  This variable seems more 'important' than some of the other included variables
3. The new documentation in this PR is specific for Apache (www-data, FcgidInitialEnv), but maybe add a comment for NGINX to look at the Apache HOME environment variable configuration?

**Comment by DelazJ** [comment]
ping! Can someone versed in Server details address, if valid, @anneb's comments please? It'd be nice to have the fix online for those that got in this kind of trouble. Thanks.

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst [hunk 1]
```diff
@@ -278,3 +278,31 @@ the data. The URL is:
 ::
 
  http://localhost/cgi-bin/project/qgis_mapserv.fcgi
+
+Cascading OGC layers
+--------------------
+
+The QGIS project we plan to use to publish WMS/WFS/WCS layers can of course
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/server/getting_started.rst]: The start of the sentence should be rewrite I think:

`A QGIS project can of course contains layers comming from ...`

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst [hunk 2]
```diff
@@ -278,3 +278,31 @@ the data. The URL is:
...
+contain layers coming from other (external) OGC servers (regardless of the underlying
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/server/getting_started.rst]: `from remote OGC servers`

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst [hunk 3]
```diff
@@ -278,3 +278,31 @@ the data. The URL is:
...
+OGC server software used). This way we will effectively **cascade** those layers
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/server/getting_started.rst]: `This way QGIS will effectively `

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst [hunk 4]
```diff
@@ -278,3 +278,31 @@ the data. The URL is:
...
+Cascading OGC layers
+--------------------
+
+The QGIS project we plan to use to publish WMS/WFS/WCS layers can of course
+contain layers coming from other (external) OGC servers (regardless of the underlying
+OGC server software used). This way we will effectively **cascade** those layers 
+through our OGC (QGIS Server based) services.
+
+If the external OGC layers are coming from services that make use of the **HTTPS**
+protocol we must take care of some extra QGIS Server configuration. Example for
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/server/getting_started.rst]: `we` -> `you`

**Comment by yjacolin** [comment]
@anneb could you change the requested changes then we can merge your PR. Thanks!

**Pull request merged**